grpc/oob/common.go

```diff
@@ -106,3 +111,15 @@ func (a *Accessory) SendStatusMessage(msg string) {
 		a.Log.V(1).Info("timed out waiting for status message receiver", "statusMsg", msg)
 	}
 }
+
+// BmcTimeoutFromCtx returns the time remaining in the context deadline.
+//
+// The returned value defaults to DefaultBmcTimeout.
+func BmcTimeoutFromCtx(ctx context.Context) time.Duration {
```

Comment:
should be `BMCTimeoutFromCtx` since BMC is an acronym and it also matches other functions in repo (e.g. grpc/oob/bmc/NewBMC ... )